cli/flox/doc/flox-upgrade.md

```diff
@@ -7,20 +7,50 @@ header: "flox User Manuals"
 
 # NAME
 
-flox-upgrade - upgrade packages using their most recent flake.
+flox-upgrade - upgrade packages in an environment
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] upgrade [ `<options>` ] `<package>` [ `<package>` ... ]
+```
+flox [ <general-options> ] upgrade
+     [-d=<path> | -r=<owner/name>]
+     [<package or group>]...
+```
 
 # DESCRIPTION
 
-Upgrade package(s) in environment.
+Upgrade packages in an environment to versions present in the environment's base
+catalog.
+
+An upgrade should usually be run after updating an environment's base catalog with
+[`flox-update(1)`](./flox-update.md).
+
+When no arguments are specified, all packages in the environment are upgraded.
+
+Packages to upgrade can be specified by either group name,
+or, if a package is not in a group with any other packages, it may be specified
+by ID.
+If the specified argument is both a group name and a package ID, only the group
+is upgraded.
```

Comment:
```suggestion
Packages to upgrade can be specified by either group name,
or, if a package is not in a group with any other packages,
it may be specified by ID.
If the specified argument is both a group name and a package ID,
only the group is upgraded.
```

Reply:
Resolving per other thread about sembr